frontend/src/pages/search.js

```diff
@@ -40,9 +40,10 @@ const Search = ({ repos, q }) => {
 
   useEffect(() => {
     setQuery(isValid ? form.query : '')
-  }, [form])
+  }, [form, isValid])
 
   useEffect(() => {
+    /* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Can you explain in a comment why it doesn't apply here?

Reply:
Resolved via eaf3263.

Reply:
You replied to the wrong comment. I've marked the other one as resolved.

Reply:
Great, I think `eslint-disable` will disable it for the rest of the file though.

Reply:
The  `/**/` works per block, removing a dependency from this hook
https://github.com/kitspace/kitspace-v2/blob/69e37ae343b45e852b5f9b9ac401938cc3655c15/frontend/src/pages/search.js#L41-L46
 which is above this block, result in this error

![image](https://user-images.githubusercontent.com/37152329/122398808-86d50800-cf7a-11eb-9990-d5ec2d8c3ef7.png)